coderd/inboxnotifications.go

```diff
@@ -37,7 +77,13 @@ func convertInboxNotificationResponse(ctx context.Context, logger slog.Logger, n
 		Targets:    notif.Targets,
 		Title:      notif.Title,
 		Content:    notif.Content,
-		Icon:       notif.Icon,
+		Icon: func() string {
+			if notif.Icon != "" {
+				return notif.Icon
+			}
+
+			return fallbackIcons[notif.TemplateID]
```

Comment:
We should probably check that there is a fallback icon associated with that template ID.